### PR TITLE
R21C landice history changes

### PR DESCRIPTION
**Pull request by lcandre2**
This request is associated with [PR#664](https://github.com/GEOS-ESM/GEOSgcm_GridComp/pull/664) in the GEOSgcm_GridComp.

These changes touch flx and glc collections and are associated with changes in the SURFACE grid comp and children (see 
 [PR#356](https://github.com/GEOS-ESM/GEOSgcm_App/pull/356) for points referenced below).
----------------
I introduced the following changes:
1. Added the cube sphere collection for glc and modified the lat/lon collection name.

2. Added 13 additional variables to the glc collection. In instances where the variable name could be confused with other variables from different collections, the name has an appended ‘GLC’. This point also includes two modifications to long names/description of landice specific outputs in the grid comp.

3. Added ‘RUNOFF’ from ‘SURFACE’ to the flx collections. This variable exports the combined land and landice runoff. I also proposed the name ‘RUNOFFTOT’  to distinguish it from ‘RUNOFF” which is the variable name for the ‘RUNSURF’ export in the lnd and other collections. This addresses point 2 in #PR356. Note that other collections use RUNSURF for RUNOFF and it is unclear if ‘RUNOFF/RUNOFFTOT’ should be used.

4. Added ‘GHTSKIN’ from ‘SURFACE’ to the flx collections and proposed the name ‘GHTFLX’. There is a landice grid comp modification to correct a sign issue and support this addition.

5. Added ‘ALBNF’ and ‘ALBVF’ from ‘SURFACE’ to the rad collections and proposed the names ‘REFLNIRDF’ and ‘REFLVISDF’ to distinguish them from the ‘SOLAR’ exports. This addresses issues with the ‘SOLAR’ exports and point 4 in #PR356. This change also includes a modification to the long name/description (albedo —> reflectivity) in the SURFACE grid comp and children (Land, Landice, Saltwater, etc).

6. Changed the long names/descriptions for AICE/OFRACI and FRSEAICE/FRACI to

Outstanding issues:
1. Note that multiple collections use RUNSURF for RUNOFF and it is unclear if RUNOFF/RUNOFFTOT should be used. A model run during NH summer would be useful to confirm that RUNOFF is exporting the combined runoff correctly. It currently does in S2S. Will confirm with a run.

2. DRPAR & DFPAR outputs from SOLAR vs SURFACE. Are they the same - will confirm with a run.

Let me know if any issues arise or if there are any questions or comments.

### CODE REVIEW

File: HISTORY_R21C.rc.tmpl [hunk 1]
```diff
@@ -1097,6 +1101,8 @@ COLLECTIONS: 'asm_inst_3hr_glo_L@HIST_IMx@HIST_JM_p48'
                                                     'ALBNR'   , 'SOLAR'    , 'ALBNIRDR'     , # confirm
                                                     'ALBVF'   , 'SOLAR'    , 'ALBVISDF'     , # confirm
                                                     'ALBVR'   , 'SOLAR'    , 'ALBVISDR'     , # confirm
+                                                    'ALBNF'   , 'SURFACE'  , 'RELFNIRDF'     , # additional alb from surface
```
Comment (gmao-rreichle) [inline HISTORY_R21C.rc.tmpl:1104]: Typo?  Should this be `REFLNIRDF`?  Same comment applies to line 1105.

**Comment by sanAkel** [comment]
> It currently does in S2S. Will confirm with a run.

Please run with the code/build from "develop" (not from S2S v3). 🙏 If need assistance don't hesitate to ask.

**Comment by sanAkel** [comment]
>This request is associated with https://github.com/GEOS-ESM/GEOSgcm_GridComp/pull/664 in the GEOSgcm_GridComp.

Add `DNA` for @sdrabenh

File: HISTORY_R21C.rc.tmpl (hunk 1 above)
Comment (lcandre2) [reply HISTORY_R21C.rc.tmpl]: Yes. Resolved in both places.

File: HISTORY_R21C.rc.tmpl [hunk 2]
```diff
@@ -1148,6 +1154,8 @@ COLLECTIONS: 'asm_inst_3hr_glo_L@HIST_IMx@HIST_JM_p48'
                                                     'ALBNR'   , 'SOLAR'    , 'ALBNIRDR'     ,
                                                     'ALBVF'   , 'SOLAR'    , 'ALBVISDF'     ,
                                                     'ALBVR'   , 'SOLAR'    , 'ALBVISDR'     ,
+                                                    'ALBNF'   , 'SURFACE'  , 'RELFLNIRDF'    , # additional alb from surface
```
Comment (gmao-rreichle) [inline HISTORY_R21C.rc.tmpl:1157]: there's still a typo here: should be REFL... not RELFL...

Comment (lcandre2) [reply HISTORY_R21C.rc.tmpl]: Actually fixed (!) in feature/rreichle/r21c_updates

**Comment by lcandre2** [comment]
Closing this pull request because I merged 
feature/lcandre2/r21c_landice_updates into feature/rreichle/r21c_updates. 
Now see [PR#356](https://github.com/GEOS-ESM/GEOSgcm_App/pull/356)

**Pull request closed without merging**